src/components/side_panel/find_and_replace/find_and_replace_store.ts

```diff
@@ -318,26 +315,31 @@ export class FindAndReplaceStore extends SpreadsheetStore {
       const zone = positionToZone(match);
       const zoneWithMerge = this.getters.expandZone(sheetId, zone);
 
-      const { x, y, width, height } = this.getters.getVisibleRect(zoneWithMerge);
+      const { width, height } = this.getters.getVisibleRect(zoneWithMerge);
       if (width > 0 && height > 0) {
-        ctx.fillStyle = BACKGROUND_COLOR;
-        ctx.fillRect(x, y, width, height);
-        if (index === this.selectedMatchIndex) {
-          ctx.strokeStyle = BORDER_COLOR;
-          ctx.strokeRect(x, y, width, height);
-        }
+        highlights.push({
+          sheetId,
+          zone: zoneWithMerge,
+          color: BORDER_COLOR,
```

Comment:
BACKGROUND_COLOR changed to BACKGROUND_COLOR ? Was the change done on purpose ?

Reply:
BACKGROUND_COLOR was the same as BORDER_COLOR + an alpha. Since the alpha is set inside the highlight, there's no need for both colors to exist

Reply:
The variable could be renamed then.
Also, the background is different before/after (the alpha is different I guess). We could use the `fillAlpha` you introduce in https://github.com/odoo/o-spreadsheet/pull/3639